pkg/operator/configobserver/node/observe_latency_profile.go

```diff
@@ -56,42 +53,37 @@ func (l *latencyProfileObserver) observeLatencyProfile(
 	existingConfig map[string]interface{},
 ) (ret map[string]interface{}, errs []error) {
 
-	// traverse list of latencyConfigs to get all the config paths
-	configPaths := make([][]string, len(l.latencyConfigs))
-	for i, latencyConfig := range l.latencyConfigs {
-		configPaths[i] = latencyConfig.ConfigPath
-	}
-
 	defer func() {
 		// Prune the observed config so that it only contains fields specific to this observer
-		ret = configobserver.Pruned(ret, configPaths...)
+		ret = configobserver.Pruned(ret, l.usedConfigPaths...)
 	}()
 
+	for _, shouldSupressConfigUpdatesFn := range l.shouldSuppressConfigUpdatesFuncs {
+		suppress, reason, err := shouldSupressConfigUpdatesFn()
+		if err != nil {
```

Comment:
also combining errs of all suppressors might be worth to consider (for the klog message)

Reply:
but we don't have to complicate it if we put the more important one (extreme profile) first it should be enough I guess

Reply:
> but we don't have to complicate it if we put the more important one (extreme profile) first it should be enough I guess

Alright, guess that's reasonable as in the operator we want to always have the rejection suppressor put first.

FYR: https://github.com/openshift/cluster-kube-controller-manager-operator/pull/629/files#diff-6c6668067473c2d5cfe0007d509b3cb098b7816b9e9f2a75268798ef5d058bf3R129-R132